### PR TITLE
HG-721 - clean up gulp tasks

### PR DESCRIPTION
**Pull request by katanka**

### CODE REVIEW

**Comment by hakubo** [comment]
This PR has more merges inside that actual changes introduced... :)
also we are considering moving to broccoli does it make sense to update those tasks?

**Comment by rogatty** [comment]
This is just adding comments and renaming of a single task, so not really "updating those tasks" ;)
Would be nice to have it in one or two commits though.

LGTM anyway.

File: gulp/tasks/default.js [hunk 1]
```diff
@@ -1,3 +1,8 @@
+/*
+ * default
+ * The default build task
```
Comment (lizlux) [inline gulp/tasks/default.js]: Might be helpful to add a note that the `default` task is what gets run when just calling `gulp` by itself.

File: gulp/tasks/docs-front.js [hunk 2]
```diff
@@ -1,3 +1,8 @@
+/*
+ * docs-front
+ * Creates documentation for front scripts
+ */
```
Comment (lizlux) [inline gulp/tasks/docs-front.js]: I didn't realize we had this! @katanka, do you happen to know where the docs can be found? Might be helpful to put that info here.

**Comment by lizlux** [comment]
LGTM

**Pull request merged**